src/styles/blocks/calendar.scss

```
@@ -0,0 +1,48 @@
+.calendar {
+  width: calendarWidth(100, 1, 10) + px;
+  display: flex;
+  flex-wrap: wrap;
+  justify-content: flex-start;
+  align-items: center;
+  gap: $cell-gap;
+  
+
+  &__cell {
```

```suggestion
  gap: $cell-gap;
  
  &__cell {
```
One empty line is enough